server/models/list.model.js

```diff
@@ -22,4 +23,28 @@ const ListSchema = new Schema(
   { timestamps: { createdAt: 'created_at' }, collection: 'lists' }
 );
 
+ListSchema.method(
```

Comment:
I think this was a bad idea to create instance's method (**I know that was my idea an my fault**) because in this case we don't operate on specific list's document (you have to create new one to use it)... Code after change looks strange. Maybe it would be better to move this 'helper' functions to utilities....